code/modules/projectiles/projectile.dm

```diff
@@ -11,6 +11,10 @@
 	pass_flags = PASSTABLE
 	mouse_opacity = MOUSE_OPACITY_TRANSPARENT
 	movement_type = FLYING
+	bound_height = 16
+	bound_width = 16
+	bound_x = 8
+	step_size = 16 // half a tile zoomer

```

Comment:
Probably should remove this line and keep them tile based until projectiles are refactored entirely